src/main/java/org/benetech/servicenet/service/impl/DocumentUploadServiceImpl.java

```diff
@@ -175,8 +179,13 @@ private DocumentUploadDTO importDataIfNeeded(String providerName, String parsedD
 
         DataImportReport reportToSave = adapter
             .map((a) -> {
+                long startTime = System.currentTimeMillis();
+                log.info("Data upload for " + providerName + " has started");
                 DataImportReport importReport = a.importData(
                     new SingleImportData(parsedDocument, report, providerName, isFileUpload));
+                long stopTime = System.currentTimeMillis();
```

Comment:
please add TODO for removing this logic in the future, you cloud also add a link to an issue for refactoring the logs.